torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1398,6 +1398,12 @@ def method_tests():
         ('__getitem__', torch.randn(S, S, S), (dont_convert([[0, 2, 3], [1, 3, 3],
                                                              torch.LongTensor([0, 0, 2])]),), 'adv_index_var'),
         ('to_sparse', (S, S), (), '', (), (), [], lambda x: x.to_dense()),
+        ('fft.fftshift', (S,), NO_ARGS, '1d', (), (None, 0, -1)),
```

Comment:
Looks like we'll need to remove these as the internal tests are complaining yet again. I'm not too worried, however. The OSS CI has run them and the autograd formulas are unlikely to change. In the near future we'll convert them into OpInfos and test their autograds that way, too.